<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileToInquiery</class>
 <widget class="QWidget" name="ProfileToInquiery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView_inquiry">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>931</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_load_inquiry">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Načíst DB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zavřít</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vyhledat podle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>číslo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_inqr_id">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>rok</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_inqr_year">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>číslo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contract_id">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>rok</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_contract_year">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_filter_inqr">
   <property name="geometry">
    <rect>
     <x>674</x>
     <y>20</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_contract">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_inqr">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_profile_manual">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Zadat číslo profilu manuálne</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_manual_profile">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_assign_inquiry">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>370</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Přiřadit k poptávce</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>370</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Sine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Random</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Shock</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Typ vibr. profilu:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> poptávky</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>index</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_inqr_index">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>zakázky</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>ProfileToInquiery</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>695</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
